docs/content/Getting-Started/user-guide.md

```diff
@@ -1,11 +1,104 @@
+
+### Usage overview
+
+The main activities in using KubeStellar MCCM are as follows.
+
+- Creating WECs (which is outside the scope of KS MCCM).
+- Bootstrapping WECs.
+- Maintaining inventory of WECs.
+- Maintaining workload descriptions in WDSes.
+- Maintaining MCCM control objects in WDSes.
+- Observing results of MCCM activity.
+
+### Bootstrapping WECs
+
+See [an example in the extended example](../../Coding%20Milestones/PoC2023q1/example1/#connect-guilder-edge-cluster-with-its-mailbox-workspace), [documentation for the main command](../../Coding%20Milestones/PoC2023q1/commands/#syncer-preparation-and-installation), and [documentation for the bundle with inventory](../../Coding%20Milestones/PoC2023q1/commands/#edge-cluster-on-boarding).
+
+### Maintaining inventory of WECs
+
+A client owns the ISes in which its inventory objects exist. The client creates and deletes ISes as desired. See [an example in the extended example](../../Coding%20Milestones/PoC2023q1/example1/#create-an-inventory-management-workspace).
+
+A client maintains inventory objects in the client's ISes. See [an example in the extended example](../../Coding%20Milestones/PoC2023q1/example1/#create-synctarget-and-location-objects-to-represent-the-florin-and-guilder-clusters), and documentation for the [postive](../../Coding%20Milestones/PoC2023q1/commands/#creating-synctargetlocation-pairs) and [negative](../../Coding%20Milestones/PoC2023q1/commands/#removing-synctargetlocation-pairs) commands.
+
+### Maintaining workload descriptions in WDSes
+
+A client owns the WDSes that it uses. The client creates and deletes WDSes as desired. See the [positive](../../Coding%20Milestones/PoC2023q1/commands/#creating-a-workload-management-workspace) and [negative](../../Coding%20Milestones/PoC2023q1/commands/#removing-a-workload-management-workspace) commands.
+
+A workload description is ordinary Kubernetes API objects. KubeStellar MCCM does not stipulate the use of any sort of "wrapper" objects. A client's workload objects may actually be some sort of wrapper, if the client has equipped its WECs with controllers that act on those wrappers. KS MCCM will transport the objects are they appear in the WDSes and WECs.
+
+### Maintaining MCCM control objects in WDSes
+
+The primary kind of control object is `EdgePlacement`. One of these goes in a WDS and binds "what" and "where". The semantics of the EdgePlacement objects are combined by OR: a workload object goes to/from a WEC if any EdgePlacement says so.
+
+There is also an API for specifying how downsynced objects may be customized on the way from the core to the WECs; see `customize.go`.
+
+### Observing results of MCCM activity
```

Comment:
I'm having trouble following the section "Observing results of MCCM activity".

Reply:
This section seems to assume a level of background understanding that is not covered in this guide.

Line 94 throws me off because we're talking about how 1:1 is the normal model, and 1:N is under development, and now we're going to talk about 1:N but where N=1. I'm getting lost here. 

"Reported state" is brought up a number of times, but the details of what these reports are is not mentioned. Is it a simple flag like "this workload is queued/running/complete", or is it something more comprehensive? What is its purpose, and where does it fit in the flowchart? I'm not sure how much of this is needed in the user guide, but I'd certainly like it detailed out in a developer guide.

As a more sweeping overview of this guide, there are a number of gritty details which appear unnecessary, but personally I've found them very informative. Ultimately these may be better served in a developer guide (or advanced usage guide), which I would look forward to once time allows one to be written.